fastmcp-auth/fastmcp_auth.py

```
@@ -0,0 +1,330 @@
+"""
+FastMCP Authentication Module
+
+This module provides JWT token generation, validation, and access token claims
+using FastMCP's BearerAuthProvider and RSAKeyPair utilities.
+"""
+
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)